.github/workflows/local.yml

```diff
@@ -55,8 +55,21 @@ jobs:
       uses: mxschmitt/action-tmate@v2
 
     - name: Build GraphScope
+      env:
+        OS : ${{ matrix.os }}
       run: |
-        /bin/bash ${GITHUB_WORKSPACE}/scripts/deploy_local.sh build_and_deploy --verbose --build_type debug
+        pushd ${GITHUB_WORKSPACE}
+        source ${HOME}/.graphscope_env
+        if [ "${OS}" == "macos-10.15" ];
+        then
+            # need libomp.dylib to find MPI_CXX
+            export LDFLAGS=-L/usr/local/opt/llvm/lib
+            make graphscope WITH_LEARNING_ENGINE=OFF NETWORKX=OFF BUILD_TYPE=debug
+        else
+            export WITH_LEARNING_ENGINE=ON
```

Comment:
why this? what's the default value?

Reply:
The default value is `ON`, but if graphscope-client packaging with graph-learn, it  need the environment `WITH_LEARNING_ENGINE` to be set `ON` .  https://github.com/alibaba/GraphScope/blob/a32e711d6dc2fd8360e5f6817fdfd38a64813719/python/setup.py#L134